deployment/stacks/without_imports/parent.ts

```diff
@@ -12,5 +13,11 @@ export class WithoutImportsParentStack extends Stack {
     props: StackProps & DeploymentConfigProperties
   ) {
     super(scope, id, props);
+
+    const {
+      stage,
+    } = props;
+
+    createDataComponents(stage);
```

Comment:
since we're only getting stage from the props, can we do props.stage and eliminate lines 17-19?